lib/typescript/render/components/RichCard/index.tsx

```diff
@@ -3,40 +3,52 @@ import styles from './index.module.scss';
 import {Media, MediaRenderProps} from '../Media';
 import {DefaultMessageRenderingProps} from '../index';
 
-interface Suggestion {
-  reply: {
-    text: string;
-    postbackData: string;
-  };
-}
+type Suggestions = [
+  {
```

Comment:
Short question: can there only be  two suggestions? Or is this possibly an array of suggestions?

Reply:
there can only be two suggestions maximum (=2 buttons) and at least one suggestion

Reply:
Ah, understood.